torchrl/envs/common.py

```diff
@@ -257,17 +255,8 @@ def step(self, tensordict: TensorDictBase) -> TensorDictBase:
         del tensordict_out
         return tensordict
 
-    def state_dict(self) -> OrderedDict:
-        return OrderedDict()
-
-    def load_state_dict(self, state_dict: OrderedDict, **kwargs) -> None:
-        pass
-
-    def eval(self) -> _EnvClass:
-        return self
-
-    def train(self, mode: bool = True) -> _EnvClass:
-        return self
+    def forward(self, tensordict: TensorDictBase) -> TensorDictBase:
+        return self.step(tensordict)
```

Comment:
forward should be undefined IMO.
Only step should be callable. Otherwise it will be confusing when we use a forward that is not step